INSTALL.md

```diff
@@ -85,7 +85,8 @@ sudo dpkg -i hush-1.0.13-afad8af-amd64.deb
 
 ## Download proving key
 ```sh
-./zcutil/fetch-params.sh
+cd ~
+../../usr/bin/./hush-fetch-params
```

Comment:
This is equivalent to `/usr/bin/hush-fetch-params` and is only correct if installing from an installed package. If you updated your change to say: "If building from source via Github, do ./zcutil/fetch-params.sh otherwise from an installed package do /usr/bin/hush-fetch-params" that would be nice to explain to end users